cmd/podman/ps.go

```diff
@@ -207,7 +207,8 @@ var (
 	}
 	psDescription = "Prints out information about the containers"
 	psCommand     = cli.Command{
-		Name:                   "ps",
+		Name:                   "list",
```

Comment:
does this need to change to list?  Couldn't it stay as ps?

Reply:
I just liked the way it looks in the help, That is really the only change.